packages/squiggle-lang/src/value/valueTags.ts

```diff
@@ -54,43 +58,25 @@ function convertToValueTagsTypeName(
 export class ValueTags {
   constructor(public value: ValueTagsType) {}
 
-  toList(): [string, Value][] {
-    const result: [string, Value][] = [];
-    const { value } = this;
-    if (value.name?.value) {
-      result.push(["name", value.name]);
-    }
-    if (value.doc?.value) {
-      result.push(["doc", value.doc]);
-    }
-    if (value.showAs) {
-      result.push(["showAs", value.showAs]);
-    }
-    if (value.numberFormat) {
-      result.push(["numberFormat", value.numberFormat]);
-    }
-    if (value.dateFormat) {
-      result.push(["dateFormat", value.dateFormat]);
-    }
-    if (value.hidden) {
-      result.push(["hidden", value.hidden]);
-    }
-    if (value.notebook) {
-      result.push(["notebook", value.notebook]);
-    }
-    const _exportData = this.exportData();
-    if (_exportData) {
-      result.push(["exportData", _exportData]);
-    }
-
-    if (value.startOpenState) {
-      result.push(["startOpenState", value.startOpenState]);
-    }
-    return result;
+  isEmpty() {
+    return (
+      this.value.name === undefined &&
+      this.value.doc === undefined &&
+      this.value.showAs === undefined &&
+      this.value.numberFormat === undefined &&
+      this.value.dateFormat === undefined &&
+      this.value.hidden === undefined &&
+      this.value.notebook === undefined &&
+      this.value.exportData === undefined &&
+      this.value.startOpenState === undefined &&
+      this.value.location === undefined
+    );
   }
 
-  isEmpty() {
-    return this.toList().length === 0;
+  toString(): string {
+    return Object.entries(this.value)
+      .map(([key, value]) => `${key}: ${value?.toString()}`)
+      .join(", ");
```

Comment:
Could also write this out. The previous setup would also work, though it's nice to organize it in ``ValueTagsWrapper``, which can do a better job. (It can convert to Values, for use with the Library)